internal/lsp/completion.go

```diff
@@ -34,6 +34,9 @@ func CompletionItem(candidate lang.CompletionCandidate, pos hcl.Pos, snippetSupp
 			Kind:             lsp.CIKField,
 			InsertTextFormat: lsp.ITFSnippet,
 			Detail:           candidate.Detail(),
+			// TODO: deprecated / tags?
+			// TODO: markdown handling
```

Comment:
I assume it's safe to say this will always be plaintext until SDK actually allows providers to declare anything else, right?

Reply:
I think the markdown support should be RFCd properly, discussed with the SDK team and it may require further changes in the protocol as mentioned.